core/keys/symbol_key.talon-list

```diff
@@ -0,0 +1,56 @@
+# symbol_key is for key names that should be available in command mode, but NOT during dictation.
+list: user.symbol_key
+-
+dot: .
+point: .
+quote: "'"
+question: ?
+apostrophe: "'"
+L square: [
+left square: [
+brack: [
+bracket: [
+left bracket: [
+square: [
+R square: ]
+right square: ]
+r brack: ]
+r bracket: ]
+right bracket: ]
+slash: /
+backslash: \
+minus: -
+dash: -
+equals: =
+plus: +
+grave: `
+tilde: ~
+bang: !
+down score: _
+underscore: _
+paren: (
+brace: {
+left brace: {
+curly bracket: {
+left curly bracket: {
+r brace: }
```

Comment:
We have "R" here but not "L".

Reply:
This variant was apparently never in keys.py. Added.